### ISSUE TITLE
Port parameter is not setting while creating GuzzleHttp\Psr7\Uri in factory.php

### ISSUE THREAD

**Issue by alisanalacam** [mailerlite/laravel-elasticsearch#84]
**Env**
```
AWS_ACCESS_KEY_ID=****
AWS_SECRET_ACCESS_KEY=****
AWS_REGION=eu-central-1
AWS_ELASTICSEARCH_ENABLED=true
ELASTICSEARCH_HOST=https://vpc-test-****.eu-central-1.es.amazonaws.com
ELASTICSEARCH_PORT=9200
ELASTICSEARCH_SCHEME=https
ELASTICSEARCH_USER=
ELASTICSEARCH_PASS=
```

**Usage**
```
$data = [
        'body' => [
        ],
        'index' => 'products',
        'type' => '_search',
    ];
    $return = Elasticsearch::index($data);
```

 **Response**
```
"exception" => GuzzleHttp\Exception\RequestException {#325 ▼
    -request: GuzzleHttp\Psr7\Request {#322 ▶}
    -response: null
    -handlerContext: array:30 [▼
      "errno" => 60
      "error" => ""
      "appconnect_time" => 0.0
      "url" => "https://vpc-test-****.eu-central-1.es.amazonaws.com/products/_search"
....
```
`
"url" => "https://vpc-test-****.eu-central-1.es.amazonaws.com/products/_search"
`
As it can be understood from the error output, the port is not exist

**current Factory.php**
```
$psr7Request = new \GuzzleHttp\Psr7\Request(
                        $request['http_method'],
                        (new \GuzzleHttp\Psr7\Uri($request['uri']))
                            ->withScheme($request['scheme'])
                            ->withHost($request['headers']['Host'][0]),
                        $request['headers'],
                        $request['body']
                    );
```
**Fix Factory.php**
```
$psr7Request = new \GuzzleHttp\Psr7\Request(
                        $request['http_method'],
                        (new \GuzzleHttp\Psr7\Uri($request['uri']))
                            ->withScheme($request['scheme'])
                            ->withPort($host['port'])
                            ->withHost($request['headers']['Host'][0]),
                        $request['headers'],
                        $request['body']
                    );
```
 **Fxied Response**
```
"exception" => GuzzleHttp\Exception\RequestException {#325 ▼
    -request: GuzzleHttp\Psr7\Request {#322 ▶}
    -response: null
    -handlerContext: array:30 [▼
      "errno" => 60
      "error" => ""
      "appconnect_time" => 0.0
      "url" => "https://vpc-test-****.eu-central-1.es.amazonaws.com:9200/products/_search"
....
```

have you ever seen this problem? How can fix it?

**Comment by cviebrock** [comment]
Sorry for the delay.

I've not encountered this issue before.  Could you make a PR to add this fix, and I'll merge it in?

**Comment by cviebrock** [comment]
Fixed by #105
